### PR TITLE
[Twig] Fix Twig namespaces

### PR DESCRIPTION
**Pull request by Phenix789**
Fix Twig namespaces used in documentation

### CODE REVIEW

File: templating/twig_extension.rst [hunk 1]
```diff
@@ -98,26 +98,30 @@ callable defined in ``getFilters()``::
...
                 // the logic of this filter is now implemented in a different class
-                new \Twig_SimpleFilter('price', array(AppRuntime::class, 'priceFilter')),
+                new TwigFilter('price', array(AppRuntime::class, 'priceFilter')),
             );
         }
     }
 
 Then, create the new ``AppRuntime`` class (it's not required but these classes
-are suffixed with ``Runtime`` by convention) and include the logic of the
+are suffixed with ``Runtime`` by convention), implement the ``RuntimeExtensionInterface`` and include the logic of the
```
Comment (HeahDude) [inline templating/twig_extension.rst]: Please move this change in a dedicated PR targeting 3.4 and rebase the current on 2.8. Thanks!

**Review by HeahDude — COMMENTED** [review]
Thank you for this first contribution!

**Comment by javiereguiluz** [comment]
@Phenix789 thanks for your first Symfony Docs contribution! We merged it in 2.8 and we'll merge in the other branches automatically. Thanks!
